mousestyles/classification/classification.py

```diff
@@ -61,10 +63,14 @@ def fit_random_forest(train_y, train_x, test_x,
     max_feature: list, optional
                  tuning parameter of RandomForest, which is the number of
                  features to consider when looking for the best split
+    importance_level: int, optional
+                      the minimum importance of features
     Returns
     ----------
-    DataFrame of RandomForest results based on testing strains.
-        Column: prediction strain labels
+    List of RandomForest results
+        The first element is the dataframe of prediction strain labels.
+        The second element is the list of tuples of score and important
+        features larger than the importance level.
     """
     # input validation
     if not isinstance(n_estimators, list):
```

Comment:
Here is the bug. So what if user does not give n_estimators, maxfeatures? We should check whether it is None first. If it is None, then we should check the validity.